_data/simple-icons.json

```
@@ -875,7 +875,7 @@
         {
             "title": "Apache Cassandra",
             "hex": "1287B1",
-            "source": "https://upload.wikimedia.org/wikipedia/commons/5/5e/Cassandra_logo.svg"
+            "source": "https://svn.apache.org/repos/asf/cassandra/logo/cassandra.svg"
```

```suggestion
            "source": "https://www.apache.org/logos"
```

Make sure your submitted icon is based on this updated source (the v4 version!), this is where we should pull our Apache project icons from.

---

I have redone the SVG from this source although it seems to be the same.